src/sentry/migrations/0073_migrate_alert_query_model.py

```diff
@@ -0,0 +1,77 @@
+# -*- coding: utf-8 -*-
+# Generated by Django 1.11.29 on 2020-05-06 21:19
+from __future__ import unicode_literals
+
+from datetime import timedelta
+
+from django.db import migrations, transaction
+from enum import Enum
+
+from sentry.utils.query import RangeQuerySetWrapperWithProgressBar
+
+
+class QueryAggregations(Enum):
+    TOTAL = 0
+    UNIQUE_USERS = 1
+
+
+aggregation_function_translations = {
+    QueryAggregations.TOTAL: "count()",
+    QueryAggregations.UNIQUE_USERS: "count_unique(user)",
+}
+
+
+def convert_alert_rule_to_snuba_query(alert_rule, SnubaQuery):
+    if alert_rule.snuba_query:
+        return
+
+    with transaction.atomic():
+        try:
+            environment = alert_rule.environment.all()[0]
+        except IndexError:
+            environment = None
```

Comment:
Is no environment all environments?

Reply:
Yeah, just means to not apply an environment filter at all.